### DISCUSSION TITLE
Mark rows different than active and urgent?

### DISCUSSION THREAD

**Discussion by defname** [davatorium/rofi#1783] [Q&A]
Hi,
I wrote a wifi menu for iwd (https://github.com/defname/rofi-iwd-wifi-menu) and want to give the possibility to style the known networks and the active networks differently from the rest. At the moment I mark them as urgent and active, since I couldn't find another possibility, but this isn't really consistent when using it together with other modes (e.g. red for the known networks, as it would be shown with most themes), isn't really appropriate).
Or is there maybe a possibility to make mode specific theme configurations (for script-modi)?

**Comment by DaveDavenport** [comment]
This is not a trivial one,   the active/urgent exists from the window switcher from the early days. This was made available in script/dmenu mode and have been part of the theme format from the beginning.

Extending this is not trivial, you want colors that match the theme, otherwise it might work on one  theme not the other.
For this we shoudl have  a unified way to specify a color pallet, but we don't have this and its not easy to add without breaking things.

However, as always, there are hacks you can do to work around this:
1.) Use pango markup (https://docs.gtk.org/Pango/pango_markup.html) to change the color.
2.) Use the theme override in script mode to change the urgent/active color (example here:https://github.com/davatorium/rofi/blob/next/Examples/test_script_mode_color.sh)  but this only works in git I think.

**Reply by defname** [reply]
Thanks for you fast reply!
I already tried the methods you suggested. When I set theme snippets with the theme option (or use "@import" with the theme option) I can achieve what I want when I run only my script, but in the case different modes are activated the options are overwritten for all other modes too.
So I will try some combination of "urgent" and pango markup I think.
